suggestions/templates/suggestions/suggestion_queue_list.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layout.html" %}
+{% block title %}
+  Suggestion Submission Queue
+{% endblock title %}
+{% block layout-content %}
+  {% load otis_extras %}
+  <p>Here are all the suggestion sets that haven't been checked yet.</p>
+  <table class="table">
+    <thead class="table-dark">
+      <tr>
+        <td class="text-end">#</td>
+        <td>Suggestion</td>
+        <td>Weight</td>
+        <td>Unit</td>
+        <td>Timestamp</td>
+        <td class="text-end">PK</td>
+      </tr>
+    </thead>
+    <tbody>
+      {% for suggestion in object_list %}
```

Comment:
I think it'd be nice to have `context_object_name = "suggestions"` in `views.py` so we can use `suggestions` here instead of `object_list`, which I think reads more nicely